tenzir/integration/lib/bats-tenzir/src/fixtures.bash

```diff
@@ -27,34 +27,32 @@ export_default_node_config() {
 
 bats_tenzir_initialize() {
   # Normalize the environment unless `BATS_TENZIR_KEEP_ENVIRONMENT` is set.
-  if [[ ! -n ${BATS_TENZIR_KEEP_ENVIRONMENT} ]]; then
-    unset $(printenv | grep -o '^TENZIR[^=]*' | paste -s -)
+  if [[ -z ${BATS_TENZIR_KEEP_ENVIRONMENT} ]]; then
+    unset "$(printenv | grep -o '^TENZIR[^=]*' | paste -s -)"
   fi
```

Comment:
This doesn't seem to work as expected.

Reply:
Yeah, adding the quotes was wrong. It's fixed now.